test_driver/mocks/services/storage_service_mocks.dart

```diff
@@ -1,18 +1,74 @@
 import 'dart:async';
 
+import 'package:crowdleague/actions/profile/update_profile_page.dart';
 import 'package:crowdleague/actions/redux_action.dart';
+import 'package:crowdleague/actions/storage/update_upload_task.dart';
+import 'package:crowdleague/enums/storage/upload_task_update_type.dart';
+import 'package:crowdleague/models/storage/upload_failure.dart';
 import 'package:crowdleague/services/storage_service.dart';
+import 'package:firebase_storage/firebase_storage.dart';
 import 'package:mockito/mockito.dart';
+import 'package:uuid/uuid.dart';
+
+import 'storage_upload_task_mocks.dart';
 
 class MockStorageService extends Mock implements StorageService {}
 
 class FakeStorageService extends Fake implements StorageService {
```

Comment:
There were two different FakeStorageService classes in separate files so I merged them into one.

Reply:
The other `FakeStorageService` class was in test_driver/mocks/storage_service_mocks.dart, which was deleted after merging into this file.

Reply:
sweet